Stream-Two/Unit03-Testing/challenges/challenge3/test_vending_machine.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from vending_machine import give_change, give_item_and_change
+
+
+class TestVendingMachine(unittest.TestCase):
+    """
+    Define a class that inherits from unittest which is the
+    unit testing model that ships with Python.
+    """
+
+    def test_return_change(self):
+        """
+        Method name has to start with test_ in order for it to run.
+        The assertEqual method is inherited from unittest.
+        :return:
+        """
+        self.assertEqual(give_change(.17), [.10, .05, .02])
+        self.assertEqual(give_change(.18), [.10, .05, .02, .01])
+        self.assertEqual(give_change(.04), [.02, .02])
+
+    def test_multiple_same_coins(self):
+        """test if change can be given in multiples of a coin
+        """
+        self.assertEqual(give_change(.4), [.2, .2])
+
+    def test_unavailable_item(self):
+        """if user asks for an item that's unavailable, they should not be given the item,
+        and their money should be returned
+        """
+        item, change, _ = give_item_and_change('crisps', .50)
```

Comment:
Perhaps mention in a comment the convention of using the underscore to signify an uninteresting return value, since I assume some students might not be familiar with it.